mondrian/src/main/java/mondrian/i18n/LocalizingDynamicSchemaProcessor.java

```diff
@@ -109,7 +109,25 @@ public String filter(
         return schema;
     }
 
-    private String doRegExReplacements(String schema) {
+    public String filter( String schema, Util.PropertyList connectInfo ) {
```

Comment:
You shouldn't need to create a class specific method. You are also circumventing a required call to super.filter(). This also forces you to do class casts in weird places later below. I think you should use the existing filter method.

Reply:
With solutions above this method will be integrated into "api".